bazel/envoy_library.bzl

```diff
@@ -91,7 +91,8 @@ def envoy_cc_library(
         tags = [],
         deps = [],
         strip_include_prefix = None,
-        textual_hdrs = None):
+        textual_hdrs = None,
+        **kargs):
```

Comment:
I believe that this was added to support `defines = ["ENVOY_WASM_V8"]` in `envoy_cc_library`, but since we only build with V8 (enabled by default), this change shouldn't be necessary anymore.

Reply:
It's a useful change in any case.  IMO the envoy_XXX rules shouldn't prevent use of other/future arguments.

However if you want it gone, done.